app/src/main/java/org/literacyapp/battery/BatteryReciever.java

```diff
@@ -0,0 +1,27 @@
+package org.literacyapp.battery;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.os.BatteryManager;
+import android.util.Log;
+
+
+public class BatteryReciever extends BroadcastReceiver {
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        int level = intent.getIntExtra(BatteryManager.EXTRA_LEVEL, -1);
+        int scale = intent.getIntExtra(BatteryManager.EXTRA_SCALE, -1);
+        float batteryPct = level / (float)scale;
+
+        Log.d("Battery status: ", String.valueOf(batteryPct));
+        if(batteryPct == 99) {
```

Comment:
Shouldn't this be something like `if (batteryPercentage < 10)` or `if (batteryPercentage < 5)`?